superset/sql_lab.py

```diff
@@ -353,9 +354,16 @@ def execute_sql_statements(  # pylint: disable=too-many-arguments, too-many-loca
                 logger.info("Query %s: %s", str(query_id), msg)
                 query.set_extra_json_key("progress", msg)
                 session.commit()
```

Comment:
Does this call `commit` on the `cursor`? I was thinking in https://github.com/googleapis/python-bigquery/issues/377 that we'd flush any batched queries when a `commit` was called, but it sounds like this may also not be desirable.

Reply:
Oh that's a different session with our metadata database to log the state of the queries, unrelated to the analytics DB